src/components/ReportSection/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+import { Collapsible } from '@edx/paragon';
+
+import { fetchReports } from '../../report/actions';
+import { reportSelector } from '../../report/selectors';
+
+
+export class ReportSection extends React.Component {
+  componentDidMount() {
+    if (this.props.programKey) {
+      this.props.fetchReports(this.props.programKey);
+    }
+  }
+
+  reportDashboard = props => (
+    <div className="container">
+      {props.reportData[props.programKey] &&
+        props.reportData[props.programKey].map(report => (
+          <div><a href={report.downloadUrl}>{report.name}</a></div>
+      ))}
+    </div>
+  )
+
+  render() {
+    if (this.props.isFirstSection) {
+      return (
+        <Collapsible
+          className="shadow"
+          title="Download Reports"
+          defaultOpen
```

Comment:
I thought in react we can do `defaultOpen` attribute with a conditional. Can you check documentation on this?